code/go/0chain.net/smartcontract/storagesc/sc.go

```diff
@@ -183,7 +183,7 @@ func (sc *StorageSmartContract) Execute(t *transaction.Transaction,
 		}
 
 	case "commit_connection":
-		resp, err = sc.commitBlobberConnection(t, input, balances)
+		resp, err = sc.commitBlobberConnection(t, input, balances, nil)
```

Comment:
Can we keep the signature of endpoints consistent? So `(t *transaction.Transaction, input []byte, balances cstate.StateContextI) (string error)`. 

We can use a local function, if convenient, to add the extra parameter.